services/concept_insights/ci.js

```diff
@@ -0,0 +1,422 @@
+/**
+ * Copyright 2016 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+module.exports = function (RED) {
+  var request = require('request');
+  var cfenv = require('cfenv');
+  var watson = require('watson-developer-cloud');
+
+  var username, password;
+  var service = cfenv.getAppEnv().getServiceCreds(/concept insights/i);
+
+  if (service) {
+    username = service.username;
+    password = service.password;
+  }
+
+  RED.httpAdmin.get('/watson-concept-insights/vcap', function (req, res) {
+    res.json(service ? {bound_service: true} : null);
+  });
+
+  function CorpusConfigurationNode(config) {
+    RED.nodes.createNode(this,config);
+    this.cname = config.cname;
+    this.access = config.access;
+    this.publiccorpus = config.publiccorpus;
+  }
+
+  RED.nodes.registerType("watson-concept-insights-corpus", CorpusConfigurationNode);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2392368955/issues/source?bid=3225115&fileBranchId=3228814#l41)